2  [error opening dir]
.
├── connection
│   ├── conn.py
│   ├── cread.txt
│   ├── __init__.py
│   ├── secure.py
├── database
│   ├── create_db.py
│   ├── db_service.py
│   ├── mysql_conf.py
|   |
│   ├── test.py
│   └── test.txt
├── main.py
├── otp.txt
├── struct.txt
└── util
    ├── display.py
    ├── env.py
    ├── __init__.py
    ├── otp_gen.py
        

